docs/metrics.md

```diff
@@ -122,7 +124,9 @@ fs_ops_count{fs_op="ReleaseFileHandle"} 9
 fs_ops_count{fs_op="StatFS"} 10
 # HELP fs_ops_error_count The cumulative number of errors generated by file system operations
 # TYPE fs_ops_error_count counter
-fs_ops_error_count{fs_error="no such file or directory",fs_op="LookUpInode"} 557
+fs_ops_error_count{fs_error="function not implemented",fs_error_category="function not implemented",fs_op="GetXattr"} 1
+fs_ops_error_count{fs_error="function not implemented",fs_error_category="function not implemented",fs_op="ListXattr"} 1
```

Comment:
should we have an example where fs_error_category is different from fs_error to show many to one mapping ?

Reply:
I've added a case where fs_error_category is different from fs_error. Showing the many-to-one mapping is however difficult since the errors where we have implemented grouping are a bit more difficult to reproduce.